docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx

```diff
@@ -193,7 +195,8 @@ The `GoogleAnalytics` component can be used to include [Google Analytics
 
 <AppOnly>
 
-To load Google Analytics for all routes, include the component directly in your root layout:
+To load Google Analytics for all routes, include the component directly in your root layout and pass
+in your measurement ID:
```

Comment:
Should line 212 be changed from `GA-XYZ` to `G-XYZ`?

Reply:
Good catch, updated!